distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestFederation.java

```diff
@@ -2185,11 +2232,17 @@ private void verifyEvents(
   }
 
   private void setupConnectedSources() throws IOException {
-    getServiceManager()
-        .createManagedService(
-            CSW_CONNECTED_SOURCE_FACTORY_PID,
-            getCswConnectedSourceProperties(
-                CONNECTED_SOURCE_ID, CSW_PATH.getUrl(), getServiceManager()));
+    connectedPid =
```

Comment:
This answers my question on Connected Sources. Might be out scope for this PR but would like to remove all connected source itests, maybe completely.